acme/acme/client.py

```diff
@@ -286,19 +286,18 @@ def poll(self, authzr):
         # TODO: check and raise UnexpectedUpdate
         return updated_authzr, response
 
-    def request_issuance(self, csr, authzrs):
```

Comment:
There are a couple of other places where this function is used but wasn't updated (`git grep request_issuance\(`):
- [ ] `poll_and_request_issuance`
- [ ] `example_client.py`